liberty-maven-plugin/src/main/resources/META-INF/plexus/components.xml

```diff
@@ -48,7 +48,7 @@
                                 net.wasdev.wlp.maven.plugins:liberty-maven-plugin:package-server
                             </package>
                             <pre-integration-test>
-                                net.wasdev.wlp.maven.plugins:liberty-maven-plugin:start-server
+                                net.wasdev.wlp.maven.plugins:liberty-maven-plugin:test-start-server
```

Comment:
you should also change `post-integration-test` - stop-server should be test-stop-server.